pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -76,7 +84,7 @@ func (o *OssmInstaller) processManifests() error {
 	}
 
 	for i, m := range o.manifests {
-		if err := m.processTemplate(data); err != nil {
+		if err := m.processTemplate(embeddedFiles, data, o.Name, o.Namespace); err != nil {
```

Comment:
I don't think we should be passing global variable `embeddedFiles` like this. Perhaps we should think about using this temporary folder which is writeable instead. So similar to what we've done before, but this time copying embedded assets to writeable FS first and essentially keeping the existing logic like it was before. WDYT?

Reply:
implemented here [6d85393](https://github.com/maistra/opendatahub-operator/pull/5/commits/6d85393bf74bb59c85fb9d732ee8b3cf6155db40), let me know what you think